decoder/attribute_candidates.go

```diff
@@ -35,6 +35,10 @@ func detailForAttribute(attr *schema.AttributeSchema) string {
 		detail = "Optional"
 	}
 
+	if attr.IsSensitive {
```

Comment:
not sure about this...

Reply:
Also for variable schema specifically we don't actually know (yet) whether a variable is required or optional without parsing the `default` - which we decided not to parse initially, so we should most certainly account for this eventuality here.

Reply:
You could probably create a string slice, such as `details []string` and then `strings.Join()` everything together upon return 🤔  and I wouldn't worry too much about the capital letter in `sensitive` - it's an edge case anyway and it's kinda debatable whether optional/required should be capitalized anyway in this context 🤷🏻